style.css

```diff
@@ -0,0 +1,78 @@
+body {
+  background-color: #fff;
+}
+
+.header {
+  position: absolute;
+  width: 375px;
+  left: calc(50% - 375px / 2);
+  display: flex;
+  justify-content: space-around;
+  border: 1px solid gray;
+  background-color: azure;
+}
+
+.logo,
+.hamburger {
+  padding: 10px 12px;
+}
+
+.hamburger img {
+  width: 30px;
+}
+
+.main {
+  background-image: url("./img/header-shapes\ mobile.png");
+  background-size: contain;
+  background-color: azure;
+  border: 1px solid gray;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 0;
+  position: absolute;
+  width: 375px;
+  height: 300px;
+  left: calc(50% - 375px / 2);
+  top: 46px;
+}
```

Comment:
- [ ] Your design is very different compared with the original design. 
### Your design
![Screenshot from 2022-12-02 01-41-43](https://user-images.githubusercontent.com/89228743/205195774-e4ec24e2-290e-4012-b5c8-3464f61e324e.png)

### Thee original design
![Screenshot from 2022-12-02 01-42-21](https://user-images.githubusercontent.com/89228743/205195800-22dfa988-6e71-4c00-9692-f67018c9fd0b.png)

- [ ]  To help you a bit fixing the issue, please replace everything else except the background image with this 👇️

```
  background-size: 100% 80vh;
  background-repeat: no-repeat;
  background-color: #fff;
  margin-bottom: 20px;
  border-radius: 0 0 0 100px;
  min-height: 100vh;
  box-sizing: border-box;
```

- [ ] Please before you submit for a second review make sure the everything looks the same compared to the original design.👌